src/components/assistant-code-block.tsx

```diff
@@ -70,6 +73,14 @@ const LanguageBlock = ( props: ContextProps & CodeBlockProps ) => {
 						} );
 					} }
 				></CopyTextButton>
+				{ isValidBlueprint && (
+					<OpenBlueprintButton
```

Comment:
Do you also think the name of button component is not accurate to its job? I think `CreateSiteWithBlueprintButton` or `AddSiteWithBlueprintButton` looks more appropriate.